README.rst

```diff
@@ -67,6 +67,8 @@ Let's start by defining a short sample ``schema.yml`` as below.
     models:
       - name: stg_users
         description: User records.
+        points_of_interest: Relevant records.
```

Comment:
These aren't in the dbt schema, you cannot add any custom fields in there. https://docs.getdbt.com/reference/configs-and-properties

Reply:
@gouline Thanks! I am adjusting this point. One follow-up question: Is this a best practice of this repository to start these meta definitions with `metabase.`, as the `special_type` is currently defined?

Reply:
That's correct. Anything can be added to `meta`, so to differentiate fields relating to Metabase, we prefix everything with `metabase.`.